src/components/Picker/BasePicker/index.js

```diff
@@ -11,6 +11,26 @@ class BasePicker extends React.Component {
         super(props);
 
         this.executeOnCloseAndOnBlur = this.executeOnCloseAndOnBlur.bind(this);
+        this.setDefaultValue = this.setDefaultValue.bind(this);
+    }
+
+    componentDidMount() {
+        this.setDefaultValue();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.items === this.props.items) {
+            return;
+        }
+        this.setDefaultValue();
+    }
+
+    setDefaultValue() {
+        // set default value only if number of items is one
```

Comment:
For code comments please...

1. Capitalize the first word
2. Explain _why_ something is done, not _what_ it is doing

Reply:
That's better, yeah. If I was writing it, it would probably say:

> When there is only 1 element in the selector, we do the user a favor and automatically select it for them so they don't have to spend extra time selecting the only possible value.